tests/tests/operation_tests.cpp

```diff
@@ -6770,6 +6770,10 @@ BOOST_FIXTURE_TEST_SUITE(operation_tests, clean_database_fixture)
 
             delegation = db->find<vesting_delegation_object, by_delegation>(std::make_tuple(op.delegator, op.delegatee));
             BOOST_CHECK(delegation == nullptr);
+
+            generate_blocks(db->head_block_time() + STEEMIT_CASHOUT_WINDOW_SECONDS);
+
+            BOOST_CHECK_EQUAL(db->get_account("bob").delegated_vesting_shares, ASSET_GESTS(0));
```

Comment:
Check received vesting shares value too.
And delegated vests instantly after operation.
It's good if tests precise: e.g. here it should additionally check delegated vests just before expiration, and that next block changes delegated vests value.